### PR TITLE
Remove package wmutils-core, create package wmutils.

### PR DESCRIPTION
**Pull request by periish**
Please fill out the template.

Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package
utilities for managing xorg from the commandline.
merged wmutils-core and wmutils-opt

## Installed manifest of package

(`cat /var/db/kiss/installed/<pkg_name>/manifest`)

```
/var/db/kiss/installed/wmutils/version
/var/db/kiss/installed/wmutils/sources
/var/db/kiss/installed/wmutils/manifest
/var/db/kiss/installed/wmutils/depends
/var/db/kiss/installed/wmutils/checksums
/var/db/kiss/installed/wmutils/build
/var/db/kiss/installed/wmutils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/xmrs.1
/usr/share/man/man1/xmmv.1
/usr/share/man/man1/wname.1
/usr/share/man/man1/wew.1
/usr/share/man/man1/chwn.1
/usr/share/man/man1/chwb2.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/man/man1/wtp.1
/usr/man/man1/wtf.1
/usr/man/man1/wrs.1
/usr/man/man1/wmv.1
/usr/man/man1/wmutils.1
/usr/man/man1/wmp.1
/usr/man/man1/wattr.1
/usr/man/man1/slw.1
/usr/man/man1/pfw.1
/usr/man/man1/mapw.1
/usr/man/man1/lsw.1
/usr/man/man1/killw.1
/usr/man/man1/ignw.1
/usr/man/man1/chwso.1
/usr/man/man1/chwb.1
/usr/man/man1/
/usr/man/
/usr/bin/xmrs
/usr/bin/xmmv
/usr/bin/wtp
/usr/bin/wtf
/usr/bin/wrs
/usr/bin/wname
/usr/bin/wmv
/usr/bin/wmp
/usr/bin/wew
/usr/bin/wattr
/usr/bin/slw
/usr/bin/pfw
/usr/bin/mapw
/usr/bin/lsw
/usr/bin/killw
/usr/bin/ignw
/usr/bin/chwso
/usr/bin/chwn
/usr/bin/chwb2
/usr/bin/chwb
/usr/bin/
/usr/
```

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/wmutils/build [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+cd core || exit 1
```
Comment (dylanaraps) [inline community/wmutils/build:3]: We don't need to do `|| exit` at all as we have `-e` in the shebang (set -e). The shell will do just this on its own.

**Comment by dylanaraps** [comment]
The manual directory is incorrect for some of the man pages.

File: community/wmutils/build [hunk 2]
```diff
@@ -7,3 +7,4 @@ make PREFIX=/usr DESTDIR="$1" install
 cd ../opt
 make
 make PREFIX=/usr DESTDIR="$1" install
+mv "$1/usr/man" "$1/usr/share/man"
```
Comment (dylanaraps) [inline community/wmutils/build:10]: `MANPREFIX=/share` should do it. See: https://github.com/wmutils/opt/blob/master/man/Makefile#L13

**Comment by periish** [comment]
```
/var/db/kiss/installed/wmutils/version
/var/db/kiss/installed/wmutils/sources
/var/db/kiss/installed/wmutils/manifest
/var/db/kiss/installed/wmutils/depends
/var/db/kiss/installed/wmutils/checksums
/var/db/kiss/installed/wmutils/build
/var/db/kiss/installed/wmutils/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/xmrs.1
/usr/share/man/man1/xmmv.1
/usr/share/man/man1/wname.1
/usr/share/man/man1/wew.1
/usr/share/man/man1/chwn.1
/usr/share/man/man1/chwb2.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/man/man1/wtp.1
/usr/man/man1/wtf.1
/usr/man/man1/wrs.1
/usr/man/man1/wmv.1
/usr/man/man1/wmutils.1
/usr/man/man1/wmp.1
/usr/man/man1/wattr.1
/usr/man/man1/slw.1
/usr/man/man1/pfw.1
/usr/man/man1/mapw.1
/usr/man/man1/lsw.1
/usr/man/man1/killw.1
/usr/man/man1/ignw.1
/usr/man/man1/chwso.1
/usr/man/man1/chwb.1
/usr/man/man1/
/usr/man/
/usr/bin/xmrs
/usr/bin/xmmv
/usr/bin/wtp
/usr/bin/wtf
/usr/bin/wrs
/usr/bin/wname
/usr/bin/wmv
/usr/bin/wmp
/usr/bin/wew
/usr/bin/wattr
/usr/bin/slw
/usr/bin/pfw
/usr/bin/mapw
/usr/bin/lsw
/usr/bin/killw
/usr/bin/ignw
/usr/bin/chwso
/usr/bin/chwn
/usr/bin/chwb2
/usr/bin/chwb
/usr/bin/
/usr/
```
Manprefix has to be set to /usr/share/man.

**Pull request merged**